tests/test_capsule/test_capsule_operations.py

```diff
@@ -28,43 +35,52 @@ def test_capsule_creation(alices_keys):
 
 
 def test_capsule_equality():
-    one_capsule = Capsule(point_e=Point.gen_rand(),
+    params = default_params()
+
+    one_capsule = Capsule(params,
```

Comment:
Wow - big change.  

This is pretty cool I think.

But I'm left wondering: if `Capsule` is going to *require* params, does it perhaps make sense to make it private?  Is there really ever a public reason for someone to init a `Capsule`?